www/include/monitoring/objectDetails/xml/hostSendCommand.php

```diff
@@ -44,16 +44,22 @@
 
 CentreonSession::start(1);
 $centreon = $_SESSION["centreon"];
-if (!isset($_SESSION["centreon"]) || !isset($_GET["host_id"]) || !isset($_GET["cmd"]) || !isset($_GET["actiontype"])) {
+if (
```

Comment:
isset() can take multiple arguments and will return true if they are all set, 
```suggestion
if (!isset($_SESSION["centreon"],$_POST["host_id"],$_POST["cmd"],$_POST["actiontype"])) {
```